biomarkerTools/riskStratAdvanced/riskStratAdv.js

```diff
@@ -223,16 +223,40 @@ function checkInputs(ind, el){
     console.log(ind);
 
 }
+
+function randomDecimalValue() {
+    return Math.floor((Math.random() * 100) +1)/ 100;
+}
+
 function addTestData() {
+    var exampleIndependent = [];
+    var exampleContour = [];
+    var exampleFixed = [];
+    var randInterator = Math.round(Math.random() * 10) + 1;
 
     thisTool.find("#independent_dropdown").val("specificity");
     thisTool.find("#contour_dropdown").val("prevalence");
     thisTool.find("#fixed_dropdown").val("delta");
```

Comment:
Rather than finding the DOM object at runtime, would it make sense to bind the DOM object to a variable so it can be called directly?

Basically:

var RSAGlobals = {};

window.onload(function() { // or whenever the DOM is finished loading
  RSAGlobals.independent_dropdown = thisTool.find("#independent_dropdown")
});
//then later, in this part of the code you would just go
RSAGlobals.independent_dropdown.val("delta")